docker-app/qfieldcloud/core/models.py

```diff
@@ -2136,6 +2241,10 @@ def get_feedback_step_data(self, step_name: str) -> dict[str, Any] | None:
         return None
 
 
+class PackageJobQuerySet(InheritanceQuerySet):
```

Comment:
```suggestion
class PackageJobQuerySet(JobQuerySet):
```

It already inherits from `InheritanceQuerySet`. What you are missing in the child job classes is `objects = JobQuerySet.as_manager()`